src/makeRobot.js

```diff
@@ -38,7 +38,59 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return (
+        'name: '
+        + this.name
+        + ', chip version: ' + this.version
+        + ', wheels: ' + this.wheels
+      );
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(steps = 1) {
+      this.coords.y += Math.max(steps, 0);
+
+      return this;
```

Comment:
No needs to invoke Math.max. Just check if step is bigger than 0 add this value. 
Apply those recommendations for other methods as well
```suggestion
     if (step > 0) {
        this.coords.y += step;
      }
      
      return this;
```